pkg/controller/volume/reconciler/reconciler.go

```diff
@@ -109,7 +109,7 @@ func (rc *reconciler) reconciliationLoopFunc() func() {
 
 				if !attachedVolume.MountedByNode {
 					glog.V(5).Infof("Attempting to start DetachVolume for volume %q from node %q", attachedVolume.VolumeName, attachedVolume.NodeName)
-					err := rc.attacherDetacher.DetachVolume(attachedVolume.AttachedVolume, rc.actualStateOfWorld)
+					err := rc.attacherDetacher.DetachVolume(attachedVolume.AttachedVolume, true /* verifySafeToDetach */, rc.actualStateOfWorld)
```

Comment:
optional: consider this as two methods: DetachVolumeSafely() and DetachVolumeNow() - they can call into a common func with a bool argument in the back.  optional.